app/src/main/java/com/cornellappdev/android/eatery/components/WaitTimesComponent.java

```diff
@@ -116,6 +126,26 @@ public boolean onTouch(View v, MotionEvent event) {
         mWaitTimesXAxisTicks = view.findViewById(R.id.wait_time_x_axis_ticks);
         mWaitTimesXAxisLabels = view.findViewById(R.id.wait_time_x_axis_labels);
 
+        mPromptTextView = view.findViewById(R.id.prompt_text_view);
+        mLowButton = view.findViewById(R.id.low_button);
+        mMediumButton = view.findViewById(R.id.medium_button);
+        mHighButton = view.findViewById(R.id.high_button);
+
```

Comment:
i would leave a short comment here about what this block of code does, just for my sake and any new devs to the repo